packages/icon-button/index.js

```diff
@@ -0,0 +1,129 @@
+import React, {Component} from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import withRipple from '@material/react-ripple';
+import {MDCIconButtonToggleFoundation} from '@material/icon-button';
+
+const {strings} = MDCIconButtonToggleFoundation;
+
+export class IconButtonBase extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      classList: new Set(),
+      childContent: props.children,
+      [strings.ARIA_PRESSED]: props[strings.ARIA_PRESSED],
```

Comment:
As far as I can tell, you never update the ARIA_PRESSED part of this.state

So why have ARIA_PRESSED in this.state at all?

Reply:
#adapter.setAttr method updates state. It really is supposed to only update aria-pressed and aria-label according to the foundation code.